### PR TITLE
documenting MC client connections when member security is enabled

### PR DESCRIPTION
**Pull request by erosb**
Backport of #50 to 4.y

### CODE REVIEW

File: docs/modules/ROOT/pages/connecting-members.adoc [hunk 1]
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
 NOTE: If the cluster uses
 xref:imdg:clusters:advanced-network-configuration.adoc[Advanced Network],
 then the provided member address should be the client address of the member.
+
+
+== Configuring Security on the Members
+
+NOTE: this section applies only to IMDG 4.2 and newer. If you use an older version, you can skip this configuration step.
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/connecting-members.adoc:34]: NOTE: This section ...

File: docs/modules/ROOT/pages/connecting-members.adoc [hunk 2]
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
...
+
+If `security` is enabled in the member configurations, then the `management-permission` should be granted for the MC client. Also, in this case the client should authenticate itself, which means you need to set up the connection by uploading a configuration XMl (or YAML) file (see above).
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/connecting-members.adoc:36]: ... granted for the Management Center client. ...

... a configuration file (XML or YAML); see xref:ROOT:managing-clusters.adoc#creating-a-cluster-configuration-by-uploading-file[here].

File: docs/modules/ROOT/pages/connecting-members.adoc [hunk 3]
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
...
+
+Example member configuration:
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/connecting-members.adoc:38]: An example member configuration is shown below:

File: docs/modules/ROOT/pages/connecting-members.adoc [hunk 4]
```diff
@@ -27,3 +27,46 @@ one cluster with the same cluster name.
...
+        <client-authentication realm="client-realm" />
+        <client-permissions>
+            <management-permission principal="mcadmin" />
+        </client-permissions>
+    </security>
+</hazelcast>
+
+```
+
+Example client configuration:
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/connecting-members.adoc:59]: An example client configuration is shown below:

**Review by Serdaro — COMMENTED** [review]
Minor comments.

**Comment by erosb** [comment]
Thank you for the review @Serdaro , I addressed your comments.
The same PR (#50) was already merged into `main`, so I will create a new PR addressing your comments there as well.

**Pull request merged**